lib/Minz/View.php

```diff
@@ -202,36 +202,25 @@ public static function headStyle () {
 		$styles = '';
 
 		foreach(self::$styles as $style) {
-			$cond = $style['cond'];
-			if ($cond) {
-				$styles .= '<!--[if ' . $cond . ']>';
-			}
-
 			$styles .= '<link rel="stylesheet" ' .
 				($style['media'] === 'all' ? '' : 'media="' . $style['media'] . '" ') .
 				'href="' . $style['url'] . '" />';
 
-			if ($cond) {
-				$styles .= '<![endif]-->';
-			}
-
 			$styles .= "\n";
 		}
 
 		return $styles;
 	}
-	public static function prependStyle ($url, $media = 'all', $cond = false) {
+	public static function prependStyle ($url, $media = 'all') {
```

Comment:
```suggestion
	public static function prependStyle($url, $media = 'all') {
```

Let's take the opportunity to use our current whitespace convention (no space before function parenthesis). Same below